lib/plugins/inventory.js

```diff
@@ -122,17 +126,26 @@ function inject (bot, { hideErrors }) {
       })
     } else if (bot.supportFeature('useItemWithOwnPacket')) {
       bot._client.write('use_item', {
-        hand: offHand ? 1 : 0
+        hand: offHand ? 1 : 0,
+        sequence
       })
     }
   }
 
   function deactivateItem () {
-    bot._client.write('block_dig', {
+    const body = {
       status: 5,
       location: new Vec3(0, 0, 0),
       face: 5
-    })
+    }
+
+    if (bot.supportFeature('useItemWithOwnPacket')) {
+      body.face = 0
+      body.sequence = 0
```

Comment:
It is just fixed to zero. I have no idea to be honest. It doesn't change the sequence count. 

I decided to add a feature support there since I wasn't sure if the face affected anything and just wanted to be as legitimate as possible without disrupting the original face (5)'s implementation.